Update release notes to current version (#1)
xillio/xillio-api-release-notes: docs/v2.8.adoc

= Version 2.8 (21-07-2020)

> Version 2.8.0 adds https://docs.xill.io/#pagination[pagination support] to the Xillio API. With pagination, you can request subsets of results. This allows you to minimize the result set to your needs and therefore have increased performance. Connectors have to support this feature.

Reviewer: ```suggestion
Version 2.8.0 adds https://docs.xill.io/#pagination[pagination support] to the Xillio API. With pagination, you can request subsets of results. This allows you to minimize the result set to your needs and therefore have increased performance. Individual connectors may or may not support this feature.
```

> Version 2.8.0 adds https://docs.xill.io/#pagination[pagination support] to the Xillio API. With pagination, you can request subsets of results. This allows you to minimize the result set to your needs and therefore have increased performance. The connector does have to support this feature in order to use it properly.

Reviewer: ```suggestion
Version 2.8.0 adds https://docs.xill.io/#pagination[pagination support] to the Xillio API. With pagination, you can request subsets of results. This allows you to minimize the result set to your needs and therefore have increased performance. Connectors have to support this feature in order to use it properly.
```
What does ' to use it properly.' mean in the context? If it refers to the performance increase, I'd probably rephrase to make that clearer.

Author: It referred to that connectors should implement this feature to be used at all. Otherwise, you'll just get the entire result set anyway. Good catch, I guess removing 'in order to use it properly' altogether is better.

> Version 2.8.0 adds https://docs.xill.io/#pagination[pagination support] to the Xillio API. With pagination, you can request subsets of results. This allows you to minimize the result set to your needs and therefore have increased performance. The connector does have to support this feature in order to use it properly.

Author: ```suggestion
Version 2.8.0 adds https://docs.xill.io/#pagination[pagination support] to the Xillio API. With pagination, you can request subsets of results. This allows you to minimize the result set to your needs and therefore have increased performance. The connector does have to support this feature.
```
